package/upgrade/upgrade_node.sh

```diff
@@ -5,12 +5,21 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 source $SCRIPT_DIR/lib.sh
 UPGRADE_TMP_DIR=$HOST_DIR/usr/local/upgrade_tmp
 
+is_mounted()
+{
+  mount | awk -v DIR="$1" '{ if ($3 == DIR) { exit 0 } } ENDFILE { exit 1 }'
+}
+
 clean_up_tmp_files()
 {
-  if [ -n "$tmp_rootfs_mount" ]; then
+  if [ -n "$tmp_rootfs_mount" ] && is_mounted "$tmp_rootfs_mount"; then
     echo "Try to unmount $tmp_rootfs_mount..."
     umount $tmp_rootfs_mount || echo "Umount $tmp_rootfs_mount failed with return code: $?"
   fi
+  if [ -n "$target_elemental_cli" ] && is_mounted "$target_elemental_cli"; then
```

Comment:
I thought we no longer need to handle `target_elemental_cli` because this PR https://github.com/harvester/harvester/pull/4154 revert it.

Reply:
Yes, you're right! Thanks for the heads-up.